test/unit/deadline_submitter_for_blender/__init__.py

```diff
@@ -1 +1,20 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+import sys
+from unittest.mock import MagicMock
+
+# Mock out UI code to test functions that use the submitter dialog.
+mock_modules = [
+    "PySide2",
+    "PySide2.QtCore",
+    "PySide2.QtGui",
+    "PySide2.QtWidgets",
```

Comment:
PySide6? Or is this not needed with qtpy?

Reply:
Tests run fine without it so I'm led to believe it's not needed!